analyzer/src/funTest/kotlin/MavenTest.kt

```diff
@@ -101,5 +101,20 @@ class MavenTest : StringSpec() {
 
             yamlMapper.writeValueAsString(result) shouldBe expectedResult
         }
+
+        "Parent POM from Maven central can be resolved" {
+            // Delete the parent POM from the local repository to make sure it has to be resolved from Maven central.
+            val userDir = File(System.getProperty("user.dir"))
```

Comment:
This should be `user.home`, see https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html.

Reply:
Done.